src/notification/NotificationContext.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Toast from "./Toast";
+export const NotificationContext = React.createContext();
+
+const createToast = (msg, options) => {
+    const id = Math.floor(Math.random() * 101 + 1);
+    const toast = {};
+    toast.id = id;
+    toast.title = msg;
+    toast.type = options.type || "simple";
```

Comment:
if someone sends a random string, will it fallback to `simple` type?

Reply:
I'll fix this